brain/python/arrayHelpers.cpp

```diff
@@ -239,6 +239,52 @@ bp::object toNumpy( const brain::Matrix4f& matrix )
     return bp::object( bp::handle<>( array ));
 }
 
+
+bp::object frameToTuple(CompartmentReportFrame&& frame)
+{
+    return bp::make_tuple(frame.getTimestamp(), toNumpy(frame.takeData()));
+}
+
+bp::object framesToTuple(CompartmentReportFrames&& frames)
+{
+    if(frames.empty())
+        return bp::make_tuple(-1.f,std::vector<float>());
+    if(frames.size() == 1)
+        return frameToTuple(std::move(frames.front()));
+
+    std::vector<double> timestamps;
+    timestamps.reserve(frames.size());
+    for (const auto& frame : frames)
+        timestamps.push_back(frame.getTimestamp());
+
+    size_t size = frames.size() * frames[0].getData().size();
```

Comment:
assign frames[0].getData().size() to a variable to reuse it in the sentence below.